tests/conftest.py

```diff
@@ -507,7 +517,7 @@ def thread_pool_executor() -> Generator[ThreadPoolExecutor, None, None]:
 
 
 @pytest.fixture
-def interpreter(system: nidaqmx.system.System) -> BaseInterpreter:
+def interpreter(system: nidaqmx.system.system) -> BaseInterpreter:
```

Comment:
This looks like a search/replace error.

It didn't break the build because this file uses `from __future__ import annotations` (which prevents the Python parser from looking up the referenced types) and we don't run mypy or pyright on this component yet.